package.json

```diff
@@ -39,8 +39,8 @@
     "axios": "^0.27.2",
     "debug": "^4.1.1",
     "lodash": "^4.17.21",
-    "snyk-delta": "^1.8.0",
     "snyk-config": "^4.0.0",
+    "snyk-delta": "^1.5.4",
```

Comment:
this is a downgrade compare to `^1.8.0`, what should be the min version now?

Reply:
that one, sorry reverting this